contrib/cppjieba-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# limonp is dependency of cppjieba.
```

Comment:
Regarding my other comment about Fasttest, we should make the build of cppjieba dependent on whether CMake variable `ENABLE_LIBRARIES` is set.

Stealing some bits from contrib/sqids-cpp-cmake/CMakeLists.txt (the CMake build description of another non-essential third-party library), this is how that can be done:

```cmake
option(ENABLE_SQIDS "Enable sqids support" ${ENABLE_LIBRARIES})
if ((NOT ENABLE_SQIDS))
    message (STATUS "Not using sqids")
    return()
endif()
```

If we go that route, we will need to
- ifdef all code which uses cppjieba (see `USE_SQIDS` in src/Functions/sqid.cpp for an example), and
- exclude SQL tests for cppjieba from FastTest (simply add `-- Tags: no-fasttest` at the beginning of the SQL test)